### PR TITLE
Added CODEOWNERS and MAINTAINERS files

### PR DESCRIPTION
**Pull request by toddysm**
Signed-off-by: Toddy Mladenov <toddysm@gmail.com>

Folks, please review this PR at your earliest convenience. We need to get those first before we can start moving the governance documents to this repository.

### CODE REVIEW

File: CODEOWNERS
Comment (priteshbandi) [inline CODEOWNERS:3]: Looks there is some permission issue for Niaz, should we fix it ?

File: MAINTAINERS
Comment (priteshbandi) [inline MAINTAINERS:7]: Should we mention repo names here?

Comment (toddysm) [reply MAINTAINERS:7]: Good point, @priteshbandi! 

We could do that. The CODEOWNERS should automatically propagate through the repos. Though for the MAINTAINERS I don't think this will work.

File: CODEOWNERS
Comment (toddysm) [reply CODEOWNERS:3]: Yes, we should. I am expecting @NiazFK @justincormack and @SteveLasker to make a decision on the Org maintainers and we will need to fix the permissions and the Teams.

File: MAINTAINERS
Comment (FeynmanZhou) [inline MAINTAINERS:8]: I think we need to add all subproject owners to this file. This list will be used as a central place to show all Notary maintainers and will be synced to http://maintainers.cncf.io/.

Comment (toddysm) [reply MAINTAINERS:8]: The CODEOWNERS file is special in this way as it can work for all subprojects. The file does not automatically imply permission though. We still need to update the teams as per the information in the file

File: MAINTAINERS
Comment (yizha1) [inline MAINTAINERS:14]: Need a newline at end of the file

File: CODEOWNERS
Comment (yizha1) [inline CODEOWNERS:12]: Need a newline at end of the file

File: MAINTAINERS
Comment (toddysm) [reply MAINTAINERS:14]: Is it required?

File: CODEOWNERS
Comment (toddysm) [reply CODEOWNERS:12]: Is it required?

File: MAINTAINERS
Comment (toddysm) [reply MAINTAINERS:7]: Re-reading the documentation it doesn't seem that we can leverage the CODEOWNERS file in the `.github` repo for the rest of the repositories. The "Repo-Level Maintainers" in this context means the `.github` repo only.

File: CODEOWNERS
Comment (SteveLasker) [reply CODEOWNERS:12]: it’s a nit, but part of markdown formatting
There are some VS Code Markdown linters that catch this.

File: MAINTAINERS
Comment (SteveLasker) [reply MAINTAINERS:7]: sounds like a nice feature request, but bummer

File: CODEOWNERS
Comment (toddysm) [reply CODEOWNERS:12]: This is not a markdown document but I will add the new line

File: MAINTAINERS
Comment (toddysm) [reply MAINTAINERS:8]: @FeynmanZhou we need to figure out how to do that for the sub-projects. Adding more folks to this one will add them as approvals to the `.github` approval, which I think should be fine because they should also participate in the governance. We need to have the maintainers for sub-projects approved first though. Will create a new PR for those once we have those cleared.

**Review by yizha1 — DISMISSED** [review]
LGTM but IANAM

File: MAINTAINERS
Comment (toddysm) [reply MAINTAINERS]: Updated with trailing new line.

File: MAINTAINERS
Comment (toddysm) [reply MAINTAINERS:8]: @FeynmanZhou Let's file another issue in this repo to synch the maintainers list with CNCF.

**Review by SteveLasker — DISMISSED** [review]
LGTM
Welcome new maintainers!

**Review by FeynmanZhou — APPROVED** [review]
LGTM but IANAM

**Review by yizha1 — APPROVED** [review]
LGTM but IANAM

**Comment by SteveLasker** [comment]
@toddysm GitHub UX still shows errors ?

**Comment by toddysm** [comment]
@SteveLasker we will fix that after the PR is merged. This is related to the GitHub Teams membership. We will make the CODEOWNERS match the GH Team

**Review by SteveLasker — APPROVED** [review]
LGTM

**Review by iamsamirzon — APPROVED** [review]

**Review by justincormack — APPROVED** [review]

**Review by vaninrao10 — APPROVED** [review]
LGTM but IANAM

**Comment by SteveLasker** [comment]
@NiazFK, can you please review?

**Review by priteshbandi — APPROVED** [review]
LGTM

**Review by NiazFK — APPROVED** [review]
LGTM

**Pull request merged**
